src/main/kotlin/no/nav/syfo/kartleggingssporsmal/application/IKartleggingssporsmalKandidatProducer.kt

```diff
@@ -1,7 +1,13 @@
 package no.nav.syfo.kartleggingssporsmal.application
 
+import no.nav.syfo.kartleggingssporsmal.domain.KandidatStatus
 import no.nav.syfo.kartleggingssporsmal.domain.KartleggingssporsmalKandidat
+import java.time.OffsetDateTime
 
 interface IKartleggingssporsmalKandidatProducer {
-    fun send(kandidat: KartleggingssporsmalKandidat): Result<KartleggingssporsmalKandidat>
+    fun send(
+        kandidat: KartleggingssporsmalKandidat,
+        status: KandidatStatus,
+        statusTidspunkt: OffsetDateTime,
+    ): Result<KartleggingssporsmalKandidat>
```

Comment:
Er det en grunn til at man må sende disse verdiene hver for seg? 😊 Tenkte alle eksisterte på `KartleggingssporsmalKandidat`